### PR TITLE
Feature: Disable access keys when content dialogs are open

### PR DESCRIPTION
**Pull request by hishitetsu**
**Resolved / Related Issues**
- [ ] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   Closes #issue...

**Validation**
How did you test these changes?
- [X] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [ ] Are there any other steps that were used to validate these changes?

### CODE REVIEW

File: src/Files.App/UserControls/AddressToolbar.xaml.cs [hunk 1]
```diff
@@ -135,5 +142,11 @@ private void OngoingTasksActions_ProgressBannerPosted(object? _, StatusCenterIte
 				userSettingsService.AppSettingsService.ShowStatusCenterTeachingTip = false;
 			}
 		}
+
+		private void Button_AccessKeyInvoked(UIElement sender, AccessKeyInvokedEventArgs args)
```
Comment (yaira2) [inline src/Files.App/UserControls/AddressToolbar.xaml.cs:146]: Can you add a comment explaining why we have this workaround?

File: src/Files.App/UserControls/InnerNavigationToolbar.xaml.cs [hunk 2]
```diff
@@ -122,5 +123,11 @@ private void SortGroup_AccessKeyInvoked(UIElement sender, AccessKeyInvokedEventA
 			}
 
 		}
+
+		private void AppBarButton_AccessKeyInvoked(UIElement sender, AccessKeyInvokedEventArgs args)
+		{
+			if (VisualTreeHelper.GetOpenPopupsForXamlRoot(MainWindow.Instance.Content.XamlRoot).Any())
```
Comment (hishitetsu) [inline src/Files.App/UserControls/InnerNavigationToolbar.xaml.cs:129]:
```suggestion
			// Suppress access key invocation if any dialog is open
			if (VisualTreeHelper.GetOpenPopupsForXamlRoot(MainWindow.Instance.Content.XamlRoot).Any())
```

File: src/Files.App/UserControls/AddressToolbar.xaml.cs [hunk 3]
```diff
@@ -135,5 +142,11 @@ private void OngoingTasksActions_ProgressBannerPosted(object? _, StatusCenterIte
...
+		{
+			if (VisualTreeHelper.GetOpenPopupsForXamlRoot(MainWindow.Instance.Content.XamlRoot).Any())
```
Comment (hishitetsu) [inline src/Files.App/UserControls/AddressToolbar.xaml.cs:148]:
```suggestion
			// Suppress access key invocation if any dialog is open
			if (VisualTreeHelper.GetOpenPopupsForXamlRoot(MainWindow.Instance.Content.XamlRoot).Any())
```

File: src/Files.App/UserControls/AddressToolbar.xaml.cs [hunk 4]
```diff
@@ -118,7 +119,13 @@ private void VisiblePath_LostFocus(object _, RoutedEventArgs e)
 
 		private void SearchRegion_OnGotFocus(object sender, RoutedEventArgs e) => ViewModel.SearchRegion_GotFocus(sender, e);
 		private void SearchRegion_LostFocus(object sender, RoutedEventArgs e) => ViewModel.SearchRegion_LostFocus(sender, e);
-		private void SearchRegion_AccessKeyInvoked(UIElement sender, AccessKeyInvokedEventArgs args) => sender.Focus(FocusState.Keyboard);
+		private void SearchRegion_AccessKeyInvoked(UIElement sender, AccessKeyInvokedEventArgs args)
+		{
+			if (VisualTreeHelper.GetOpenPopupsForXamlRoot(MainWindow.Instance.Content.XamlRoot).Any())
```
Comment (hishitetsu) [inline src/Files.App/UserControls/AddressToolbar.xaml.cs:124]:
```suggestion
			// Suppress access key invocation if any dialog is open
			if (VisualTreeHelper.GetOpenPopupsForXamlRoot(MainWindow.Instance.Content.XamlRoot).Any())
```

**Review by yaira2 — APPROVED** [review]
LGTM

**Pull request merged**